Make MAVLink site more implementer-agnostic

hamishwillee opened the pull request:
As discussed in MAVLink dev meeting, we want to make sure the mavlink site is open and unbiased. This PR implements the changes discussed in the call. 
- Header includes no links to specific implementations (GCS/Autopilots)
- Sidebar includes no links to specific implementations
- Header has links to key resources - including "Support" which now has its own page.
- Governance section added to home page, which shows that Dronecode and LF host the project
- Logo in top left now shows both mavlink and dronecode.

You can preview [here](https://hamishwillee.gitbooks.io/ham_mavdevguide/content/v/header/en/). Note however that clicking the top links will take you to the *current* site (hard coded, so will work once this is merged.)

khancyr commented on kr/about/implementations.md:
```diff
@@ -0,0 +1,46 @@
+* [Dronecode SDK](https://sdk.dronecode.org/en/) - MAVLink API Library for the Dronecode Platform (cross platform, optimised for PX4).
+* [Dronecode Camera Manager](https://camera-manager.dronecode.org/en/) - Adds [Camera Protocol](../protocol/camera.md) interface for cameras connected to Linux computer
+* [Rosetta Drone](https://www.youtube.com/watch?v=rBqEQoVGuzQ) - MAVLink wrapper around DJI SDK (fly a DJI drone with a Mavlink-based GCS, code: https://github.com/diux-dev/rosettadrone)
+* [pymavlink](https://github.com/mavlink/pymavlink) - MAVLink python bindings
```
https://github.com/ArduPilot/pymavlink as mavlink fork doesn't seems to exist anymore

hamishwillee replied:
Thanks very much. Fixed!

auturgy commented:
Thanks Hamish. A good step forward.

auturgy commented:
As discussed: There are a few things that should be fixed: qgc actually defaults to ardupilotmega.xml, not common, a few gcs’s are missing and dronekit should be added to the api’s, but I thought that could wait. (Qgc probably doesn’t need to be listed 5 times either, unless every flavour of tower and mission planner is going to be added too).

mrpollo commented:
@hamishwillee I think its better if we remove the dronecode wings from the top bar

hamishwillee commented:
Thanks @auturgy .

- I added the new flight stacks.
- I compacted the QGC variants into one line as I don't think it is scalable or reasonable to given variants equal billing .
- Can you supply Mission Planner variants in a separate PR?
- Added DroneKit to list

hamishwillee commented:
OK, made some further changes - the main one being that this now has only MAVLink logo up to left of header. 

I am going to merge this now. Obviously open to further feedback and PRs.
